qa/automation/CareerPortal.Tests.Scenarios/Scenarious/All_Buttons.feature

```diff
@@ -0,0 +1,44 @@
+Background:
+Given I open the https://career.quantori.com/ portal in Google Chrome
+
+Scenario 1.0: Cookie Test
+When I click on "Accept and Continue" button in pop-up cookie window 
+And click to the browser Settings/Privacy and Security/Cookies and other site data/See all cookies
+Then I should see the new cookie file from portal was added
+
+Scenario 1.0.1: Cookie Test
+When I click on "Learn more" button in pop-up cookie window 
+Then I should see the pop-up window about Cookies and Data Privacy Policy
+
+Scenario 1.1: "Search" button
+When I click on "Search" button
+Then I should see open job positions for default Competence and Technology
```

Comment:
You should list the vacancies which are available and as this list may change from time to time, you shold mention in Background that specific positions are open and available. When you have such sepcific background, in this test you can list these vacancies.